java/server/src/org/openqa/selenium/grid/console/src/assets/edge-new.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" width="48px" height="48px"><path fill="#1de9b6" d="M4,23.481C4,15.673,10.557,4,24.014,4C37.452,4,44,13.923,44,20.905c0,5.604-4.429,9.732-9.992,9.732	c-4.245,0-6.759-1.408-6.759-2.53c0-0.726,1.755-1.837,1.755-3.982c0-2.565-1.986-5.137-5.045-5.137	c-2.934,0-4.974,2.442-4.974,4.905c0,5.952,5.63,11.644,13.931,11.644c4.959,0,6.759-1.828,7.403-1.828	c0.196,0,0.647,0.08,0.647,0.603c0,0.5-5.486,9.686-16.94,9.686C12.632,44,4,35.023,4,23.481z"/><radialGradient id="2pm2zgLwrfHW4PMfqfOjYa" cx="17.993" cy="32.825" r="21.348" gradientTransform="matrix(1 0 0 1.1083 0 -3.529)" gradientUnits="userSpaceOnUse"><stop offset=".652" stop-color="#0083de"/><stop offset=".822" stop-color="#006aac"/><stop offset=".944" stop-color="#005a8c"/></radialGradient><path fill="url(#2pm2zgLwrfHW4PMfqfOjYa)" d="M4,23.481c0-7.808,7.615-12.04,12.73-12.04c13.439,0,17.277,2.483,17.277,9.465	c0,5.604-5.003,5.366-5.003,3.221c0-2.565-1.986-5.137-5.045-5.137c-2.934,0-4.974,2.442-4.974,4.905	c0,5.952,5.63,11.644,13.931,11.644c4.959,0,6.759-1.828,7.403-1.828c0.196,0,0.647,0.08,0.647,0.603c0,0.5-5.486,9.686-16.94,9.686	C12.632,44,4,35.023,4,23.481z"/><radialGradient id="2pm2zgLwrfHW4PMfqfOjYb" cx="28.082" cy="29.295" r="16.478" gradientTransform="matrix(1 0 0 1.0028 0 -.067)" gradientUnits="userSpaceOnUse"><stop offset=".69" stop-color="#05509b"/><stop offset=".717" stop-color="#054e98"/><stop offset="1" stop-color="#033e78"/></radialGradient><path fill="url(#2pm2zgLwrfHW4PMfqfOjYb)" d="M18.985,23.894c0-2.298,1.775-4.575,4.395-4.871c-4.104,0.349-8.798,4.375-8.798,11.831	c0,7.963,6.864,13.465,13.943,12.628c8.386-1.927,12.441-8.74,12.441-9.168c0-0.523-0.451-0.603-0.647-0.603	c-0.643,0-2.444,1.828-7.403,1.828C24.615,35.539,18.985,29.847,18.985,23.894z"/><radialGradient id="2pm2zgLwrfHW4PMfqfOjYc" cx="47.648" cy="16.079" r="61.532" gradientTransform="matrix(-1 0 0 -1.5084 95.296 40.31)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#5eeb69"/><stop offset=".214" stop-color="#4bc958"/><stop offset=".225" stop-color="#4ac862"/><stop offset=".272" stop-color="#47c68b"/><stop offset=".323" stop-color="#44c4ae"/><stop offset=".376" stop-color="#41c2cb"/><stop offset=".433" stop-color="#3fc1e1"/><stop offset=".495" stop-color="#3ec0f1"/><stop offset=".565" stop-color="#3dbffa"/><stop offset=".664" stop-color="#3dbffd"/></radialGradient><path fill="url(#2pm2zgLwrfHW4PMfqfOjYc)" d="M4,23.481C4,15.673,10.557,4,24.014,4C37.452,4,44,13.923,44,20.905	c0,5.604-4.429,9.732-9.992,9.732c-4.245,0-6.759-1.408-6.759-2.53c0-0.726,1.755-1.837,1.755-3.982	c0-2.565-2.076-10.678-12.628-10.678C13.442,13.448,5.256,14.745,4,23.481z"/></svg>
```

Comment:
Have you used an SVG compressor?

Reply:
Yeah. I did minify each one of the svgs by importing them in Adobe XD and exporting them as Optimized svgs.